persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -57,6 +63,13 @@ private PersistToolsConstants() {
     public static final String PORT = "port";
 
     public static final String CREATE_DATABASE_SQL = "CREATE DATABASE %s";
+    public static final String KEYWORD_PROVIDER = "provider";
+    public static final String KEYWORD_SHADOWDATABASE = "shadowDatabase";
+    public static final String SUBMODULE_PERSIST = "persist";
+    public static final String SUBMODULE_MODULES = "modules";
+    public static final String KEYWORD_CLIENTS = "clients";
+    public static final String PERSISTTOML = "Persist.toml";
```

Comment:
```suggestion
    public static final String PERSIST_TOML_FILE = "Persist.toml";
```

Reply:
Updated in [d52eb35](https://github.com/ballerina-platform/persist-tools/pull/52/commits/d52eb354511a981115450feea7522cfe08f7c1c7)